comment_generation/app.py

```
@@ -0,0 +1,88 @@
+            "headers": headers,
+            "body": json.dumps({"error": "Prompt is required"}),
+        }
+
+    try:
+        response = client.responses.create(
+            model="gpt-5-nano",
+            input=prompt,
+            instructions="あなたはユーザーの友人で、ユーザーと一緒にDLsiteを見ています。"
+        )
+
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)